packages/ses/README.md

```diff
@@ -689,7 +695,11 @@ report available on the
 [Agoric blog](https://agoric.com/blog/technology/metamask-agoric-hardened-js-security-review/)
 that includes links to recordings of code walk-throughs and technical
 discussion, and issues are tagged
-[audit-SEStival](https://github.com/endojs/endo/labels/audit-sestival). 
+[audit-SEStival](https://github.com/endojs/endo/labels/audit-sestival).
```

Comment:
Since you're citing it as history, you should include the closed bugs as well.

```suggestion
[audit-SEStival](https://github.com/endojs/endo/issues?q=label%3Aaudit-sestival+).
```